docs/sources/flow/reference/components/otelcol.exporter.otlphttp.md

```diff
@@ -0,0 +1,205 @@
+---
+aliases:
+- /docs/agent/latest/flow/reference/components/otelcol.exporter.otlphttphttp
+title: otelcol.exporter.otlphttphttp
+---
+
+# otelcol.exporter.otlphttp
+
+`otelcol.exporter.otlphttp` accepts telemetry data from other `otelcol`
+components and writes them over the network using the OTLP HTTP protocol.
+
+> **NOTE**: `otelcol.exporter.otlphttp` is a wrapper over the upstream
+> OpenTelemetry Collector `otlphttp` exporter. Bug reports or feature requests
+> will be redirected to the upstream repository, if necessary.
+
+Multiple `otelcol.exporter.otlphttp` components can be specified by giving them
+different labels.
+
+## Usage
+
+```river
+otelcol.exporter.otlphttp "LABEL" {
+  client {
+    endpoint = "HOST:PORT"
+  }
+}
+```
+
+## Arguments
+
+`otelcol.exporter.otlphttp` supports the following arguments:
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`metrics_endpoint` | `string` | The endpoint to send metrics to | `client.endpoint + "/v1/metrics"` | no
+`logs_endpoint`    | `string` | The endpoint to send logs to    | `client.endpoint + "/v1/logs"`    | no
+`traces_endpoint`  | `string` | The endpoint to send traces to  | `client.endpoint + "/v1/traces"`  | no
+
+The default value depends on the `endpoint` field set in the required `client`
+block. If set, these arguments override the `client.endpoint` field for the
+corresponding signal.
+
+## Blocks
+
+The following blocks are supported inside the definition of
+`otelcol.exporter.otlphttp`:
+
+Hierarchy | Block | Description | Required
+--------- | ----- | ----------- | --------
+`client`           | [client][] | Configures the HTTP server to send telemetry data to. | yes
+`client > tls`     | [tls][] | Configures TLS for the HTTP client. | no
+`queue`            | [queue][] | Configures batching of data before sending. | no
+`retry`            | [retry][] | Configures retry mechanism for failed requests. | no
```

Comment:
We deliberately don't use code fonts in the hierarchies because it looks weird. [Example](https://grafana.com/docs/agent/latest/flow/reference/components/discovery.kubernetes/#blocks)

```suggestion
client           | [client][] | Configures the HTTP server to send telemetry data to. | yes
client > tls     | [tls][] | Configures TLS for the HTTP client. | no
queue            | [queue][] | Configures batching of data before sending. | no
retry            | [retry][] | Configures retry mechanism for failed requests. | no
```